### PR TITLE
Adding support for `kubernetesServiceIpAddressRange` to `eks.Cluster`

### PR DESCRIPTION
**Pull request by stack72**
Fixes: #443

### CODE REVIEW

File: examples/examples_nodejs_test.go [hunk 1]
```diff
@@ -47,6 +47,22 @@ func TestAccCluster(t *testing.T) {
 	integration.ProgramTest(t, &test)
 }
 
+func TestAccKubernetesServiceIPv4RangeForCluster(t *testing.T) {
```
Comment (justinvp) [inline examples/examples_nodejs_test.go:50]: Now that we have Python and .NET hooked up in CI, it'd be nice to add tests for those languages as well (and Go coming soon). I can help with that as a follow-up, if you'd like.

File: nodejs/eks/cluster.ts [hunk 2]
```diff
@@ -475,6 +475,13 @@ export function createCore(name: string, args: ClusterOptions, parent: pulumi.Co
         }));
     }
 
+    let kubernetesNetworkConfig: pulumi.Output<aws.types.output.eks.ClusterKubernetesNetworkConfig> | undefined;
+    if (args.kubernetesServiceIpAddressRange) {
+        kubernetesNetworkConfig = pulumi.output(args.kubernetesServiceIpAddressRange).apply(
+            serviceIpv4Cidr => <aws.types.output.eks.ClusterKubernetesNetworkConfig>({serviceIpv4Cidr}),
+        );
+    }
```
Comment (justinvp) [inline nodejs/eks/cluster.ts:483]: Technically this should be the _input_ type, and no need to cast inside the apply:


```suggestion
    let kubernetesNetworkConfig: pulumi.Output<aws.types.input.eks.ClusterKubernetesNetworkConfig> | undefined;
    if (args.kubernetesServiceIpAddressRange) {
        kubernetesNetworkConfig = pulumi.output(args.kubernetesServiceIpAddressRange).apply(
            serviceIpv4Cidr => ({ serviceIpv4Cidr })
        );
    }
```

File: examples/cluster-with-serviceiprange/Pulumi.yaml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+name: example-cluster-with-serviceIpRange
```
Comment (justinvp) [inline examples/cluster-with-serviceiprange/Pulumi.yaml:1]: Super nitpick (sorry!):

```suggestion
name: example-cluster-with-serviceiprange
```

File: examples/cluster-with-serviceiprange/package.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{
+    "name": "example-cluster-with-serviceIpRange",
```
Comment (justinvp) [inline examples/cluster-with-serviceiprange/package.json:2]: Super nitpick:

```suggestion
    "name": "example-cluster-with-serviceiprange",
```

**Review by justinvp — APPROVED** [review]
LGTM otherwise.

File: examples/examples_nodejs_test.go (hunk 1 above)
Comment (stack72) [reply examples/examples_nodejs_test.go:50]: I will follow up with these when I get a local dev setup working for EKS in these langs

**Pull request merged**